src/shared/agent_op.c

```diff
@@ -619,8 +621,17 @@ static cJSON* w_create_agent_add_payload(const char *name,
         cJSON_AddStringToObject(arguments, "id", id);
     }
 
-    if (force >= 0) {
-        cJSON_AddNumberToObject(arguments, "force", force);
+    if (force_options) {
+        cJSON* j_force = cJSON_CreateObject();
```

Comment:
In case any of the other changes requires converting an **authd_force_options_t** structure into a json, we could create a helper method **force_options_to_json()**

Reply:
Good idea !